app/code/Magento/Backend/Block/Widget/Form/Element/Dependence.php

```diff
@@ -124,14 +124,17 @@ protected function _toHtml()
         if (!$this->_depends) {
             return '';
         }
-        return '<script>
-            require(["mage/adminhtml/form"], function(){
-        new FormElementDependenceController(' .
-            $this->_getDependsJson() .
-            ($this->_configOptions ? ', ' .
-            $this->_jsonEncoder->encode(
-                $this->_configOptions
-            ) : '') . '); });</script>';
+
+        $dependsJson = $this->_getDependsJson();
+        $configOptions = $this->_configOptions ? $this->_jsonEncoder->encode($this->_configOptions) : null;
+        $configOptionsStr = $configOptions ? ', ' . $configOptions : '';
+        $paramsStr = $dependsJson . $configOptionsStr;
```

Comment:
IMO readability could be slightly improved if you do like this:
```
$params = $this->_getDependsJson();
if ($this->_configOptions) {
    $params .= ', ' . $this->_configOptions;
}
```

Reply:
@orlangur, Absolutely agree with you. I updated pull request according to your suggestion.